src/context/QueryProvider.tsx

```diff
@@ -1,16 +1,32 @@
 import type { ReactNode } from 'react'
 import { QueryClient, QueryClientProvider } from '@tanstack/react-query'
+import { createSyncStoragePersister } from '@tanstack/query-sync-storage-persister'
+import { persistQueryClient } from '@tanstack/react-query-persist-client'
 
 type TQueryProviderProps = {
   children: ReactNode
 }
 
-const queryClient = /* #__PURE__ */ new QueryClient()
+const queryClient = /* #__PURE__ */ new QueryClient({
+  defaultOptions: {
+    queries: {
+      cacheTime: 1000 * 60 * 60 * 24 * 7, // 1 week
+      staleTime: 1000 * 60 * 60 * 2 // 2 hours
+    }
+  },
+})
```

Comment:
We use `react-query` in `useBalances` as well. With such settings it might be affected. Have you checked it?

Reply:
All balances queries has own `refetchInterval`